### DISCUSSION TITLE
Crowdsec struggles

### DISCUSSION THREAD

**Discussion by chief-lewus** [ZoeyVid/NPMplus#1075] [Q&A] [answered]
I've attempted a few times now to get Crowdsec working and I have failed multiple times. 

Here is my compose file. I have tried Crowdsec as Network mode Host and Network mode Bridge and the same issue persists.
![NPMPLUS-COMPOSE](https://github.com/user-attachments/assets/662e5030-326a-4d67-8f42-3c5f598a09dd)

Here is my npmplus.yaml file that I have placed inside /proxy/crowdsec/conf/acquis.d/npmplus.yaml
![NPM-ACQUIS](https://github.com/user-attachments/assets/4692381f-dcb8-4cc8-9884-a09179ae79b2)

And here is my Crowdsec config with my bouncer and set to enabled. Located at /proxy/npm/etc/crowdsec
![CONF](https://github.com/user-attachments/assets/ce10d9f6-9527-4785-9a98-53855142274d)

Things I've tried... Making a custom whitelist for my local IP's, Setting Crowdsec container to Host and Bridge. While crowdsec is enabled I get an access forbidden when I try to access my sites. If I check the active decisions list there is nothing active nor are the any alerts when checked there.

**Comment by Zoey2936** [comment]
You need to create the api key as per readme

**Reply by chief-lewus** [reply]
I did create the key with the command in the readme and placed it in the config. I just had it blacked out in the image

**Reply by Zoey2936** [reply]
what failed exactly? any logs? errors?

**Reply by chief-lewus** [reply]
I wouldn't say anything failed. I just keep getting the Crowdsec Forbidden Access page when I have it enabled, despite having made a custom whitelist for my local IP's and such. If I check the Crowdsec decisions it doesn't show anything as banned either.

**Reply by Zoey2936** [reply]
Is there something written in npmplus docker logs? It looks like it maybe can't connect to crowdsec

**Reply by chief-lewus** [reply]
I'm going to try a fresh install again and see if something just didn't click the first time or I overlooked something. 

If I enable Crowdsec and I try to visit a site I just get the "Access Forbidden Crowdsec" page. As soon as I disable it, All is well again. I've tried whitelisting to no avail.

**Reply by Zoey2936** [reply, marked as answer]
Yes, please send logs

**Reply by mshihadeh1** [reply]
I have the same issue as the person in the initial post. I have tried following the directions as per readme and referencing the .yaml. I have also tried following this [guide](https://geekscircuit.com/installing-crowdsec-with-nginx-proxy-manager-npmplus/).

Here are my logs:

![_npmplus_logs-1](https://github.com/user-attachments/assets/e52db941-b913-4be4-87aa-1d99ae443c23)

**Reply by Zoey2936** [reply]
the guide you mentioned is not the best, can you please create a new discussion for this, also post there your compose.yaml and all crowdsec confs

**Reply by mshihadeh1** [reply]
got it thank you
